src/flash.c

```diff
@@ -856,7 +856,7 @@ flash_write_binary (uint8_t file_id, const uint8_t *data,
   else if (file_id == FILEID_CH_CERTIFICATE)
     {
       maxsize = FLASH_CH_CERTIFICATE_SIZE;
-      if(_selected_identity==2){
+      if(_selected_identity==2){ // TODO: remove block, as it seems redundant
```

Comment:
Can confirm, it's redundant. I fixed it in the define and didn't clean up all the use points.

Reply:
Removed, thank you!